targets/simple_switch/primitives.cpp

```diff
@@ -264,8 +274,19 @@ class modify_field_with_hash_based_offset
                            const NamedCalculation &, const Data &> {
   void operator ()(Data &dst, const Data &base,
                    const NamedCalculation &hash, const Data &size) {
-    uint64_t v =
-      (hash.output(get_packet()) % size.get<uint64_t>()) + base.get<uint64_t>();
+    uint64_t b = base.get<uint64_t>();
```

Comment:
can you use `auto` for these variable declarations, the type is obvious from the `get<uint64_t>`